packages/block-editor/package.json

```diff
@@ -23,6 +23,8 @@
 	"react-native": "src/index",
 	"dependencies": {
 		"@babel/runtime": "^7.8.3",
+		"@emotion/core": "^10.0.28",
+		"@emotion/styled": "^10.0.27",
```

Comment:
Should we just keep the CSS in JS in the components package for now for consistency?

Reply:
Sure. I can refactor this 👌